src/EventStore.Projections.Core/ProjectionsSubsystem.cs

```diff
@@ -215,15 +215,15 @@ public sealed class ProjectionsSubsystem :ISubsystem,
 		public void Handle(ProjectionSubsystemMessage.RestartSubsystem message) {
 			if (_restarting) {
 				_logger.Information("PROJECTIONS SUBSYSTEM: Not restarting because the subsystem is already being restarted.");
-				message.ReplyEnvelope.ReplyWith(new ProjectionSubsystemMessage.InvalidSubsystemRestart("Restarting"));
+				message.ReplyEnvelope.ReplyWith(new ProjectionSubsystemMessage.InvalidSubsystemRestart("Restarting", "Projection Subsystem cannot be restarted as it is in the wrong state"));
```

Comment:
we have a bit more info logged above, it might be nice to include the extra info in the message. we could put the reason in a variable and use it for the logging and the reply